docs/backup-restore.md

```diff
@@ -1,32 +1,99 @@
+stop the Cassandra cluster and take the copies of all the nodes of the cluster, and start the cluster. 
 
-To minimize mistakes when doing backup and restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+To avoid mistakes when doing backup operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
 Cassy is also integrated with `scalar-admin` so it can issue a pause request to the application of a Cassandra cluster.
 Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
 
-## JDBC databases
+**Cosmos DB**
 
-You can take a backup with your favorite way for JDBC databases.
-One requirement for backup in Scalar DB on JDBC databases is that backups for all the Scalar DB managed tables (including the coordinator table) need to be transactionally-consistent or automatically recoverable to a transactionally-consistent state.
-That means that you need to create a consistent snapshot by dumping all tables in a single transaction.
-For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
-Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
+You must create a Cosmos DB account with a Continuous backup policy to create point-in-time restore (PITR).
+Please follow [the basic strategy](#general-way-to-create-a-transactionally-consistent-backup) section to create a backup.
+
+**DynamoDB**
+
+You must create tables with point-in-time recovery (PITR) and auto-scaling in DynamoDB. Scalar DB Schema Loader enables PITR and auto-scaling by default.
+Please follow [the basic strategy](#general-way-to-create-a-transactionally-consistent-backup) section to create a backup.
+
+#### Basic strategy to create a transactionally-consistent backup
+
+One way to create a transactionally-consistent backup is to take a backup while Scalar DB cluster does not have outstanding transactions. 
+If an underlying database supports a point-in-time snapshot/backup mechanism, you can take a snapshot during the period.
+If an underlying database supports a point-in-time restore/recovery mechanism, you can set a restore point to a specific time (preferably the midtime) in the period since the system takes backups for each operation in such a case.
+
+To easily achieve transactionally-consistent backup for Scalar DB on a non-transactional database is to use the [Scalar DB server](https://github.com/scalar-labs/scalardb/tree/master/server) or implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application,
```

Comment:
i think we can describe more on how we make use of scalar-admin to pause the scalar DB servers on goining transactions here.

Reply:
Thank you for your review comments.
Document updated based on review comments.